src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task {
+
+    public ToDo(String taskName) {
+        super(taskName);
+    }
+
+    @Override
+    public void getTaskStatus() {
```

Comment:
Could consider changing the name of the function to `printTaskStatus` instead since it prints directly? The word `get` implies that it would be returning something.